client/src/components/Logo/Logo.css

```diff
@@ -0,0 +1,13 @@
+.Logo {
+  background-image: url("./logo.png");
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 20vh;
+  height: 20vh;
```

Comment:
I would do this in pixels or rems - maybe like `3rem` or `4rem` for large and `2rem` for small?